openedx/core/djangoapps/content_tagging/rest_api/v1/tests/test_views.py

```diff
@@ -1534,6 +1541,129 @@ def test_tag_xblock_invalid(self, user_attr, taxonomy_attr):
         response = self.client.put(url, {"tags": ["invalid"]}, format="json")
         assert response.status_code == status.HTTP_400_BAD_REQUEST
 
+    @ddt.data(
+        # userA and userS are staff in libraryA and can tag using enabled taxonomies
```

Comment:
Who are `userA` and `userS` ? I see them mentioned a few times in this file, but I think those are typos or from an older version. Here I only see `user`, `staff`, and `staffA`.

Reply:
Done: 78b3d53b7ca1909ae83ed0472be40696de3e05b2